src/WebJobs.Script/Config/ScriptSettingsManager.cs

```diff
@@ -68,6 +68,31 @@ public virtual string AzureWebsiteUniqueSlotName
             }
         }
 
+        public virtual string ApplicationInsightsInstrumentationKey
+        {
```

Comment:
Except I think I had a bad merge and lost my app.config setting :-) -- I'll add it back in a subsequent checkin.